teps/0042-taskrun-breakpoint-on-failure.md

```diff
@@ -0,0 +1,119 @@
+---
+title: taskrun-breakpoint-on-failure
+authors:
+  - "@waveywaves"
+creation-date: 2021-01-15
+last-updated: 2021-01-15
+status: implementable
```

Comment:
before we go to "implementable" we'd need to have the proposal, alternatives, etc., so maybe we can make this "proposed" for now?

Reply:
Updated to proposed :raised_hands:

Reply:
looks like the status is now "implementing" which seems like we've jumped the gun a bit - can we get this in as proposed (or at least implementable) before that?